terraform/remote.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  backend "remote" {
+    organization = "usdr"
+
+    workspaces {
+      name = "appointment-availability-infra"
+    }
+  }
+}
```

Comment:
Now that this is here, what can or should we do to discourage running commands that might taint the production state? Do they always require manually locking the workspace, and is that clear enough?

(In reference to Slack discussion: https://usdigitalresponse.slack.com/archives/C01S4JWSMT7/p1625088086008200?thread_ts=1625024517.032900&cid=C01S4JWSMT7)